### ISSUE TITLE
Ghired, Mirror of the Wilds's ability is don't work.

### ISSUE THREAD

**Issue by stormcat664** [Card-Forge/forge#5075]
**Describe the bug**
When I tried to use Ghired, Mirror of the Wilds in the OTJ Sealed, the activated ability he gives did not work well.

**To Reproduce**
1. Control Ghired and non-token creatures that are not summoning sickness.
2. Putting creatures onto the battlefield.
3. The ability by Ghired tries to activate.

**Expected behavior**
Copying ability can be activated.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox
 - Version forge-gui-desktop-1.6.62-SNAPSHOT-04-19

**Additional context**
Basically, no options were also presented, but text was only available for saddle creatures.

**Comment by dracontes** [comment]
I'm sorry, but you're going to have provide a bit more information. I just tested the card, to be sure in `SNAPSHOT-04-18`, and it works fine. Also, if I'm allowed to nitpick wording, the way you described the issue has me wonder if you do understand what the card does: Ghired grants the activated ability to itself and the ability looks for tokens that entered this turn. 
![image](https://github.com/Card-Forge/forge/assets/45150760/1b81bb9b-0fe4-4897-aba0-1c2cf8bda088)

**Comment by stormcat664** [comment]
Oh, it certainly works now.
I made the mistake of deleting the previous version and forgot to take screenshots.
